src/components/generic/Banner/Banner.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import webexComponentClasses from '../../helpers';
+
+/**
+ * Banner component.
+ *
+ * @param {object} props  Data passed to the component
+ * @param {React.ReactNode[]} props.children  List of children
```

Comment:
i think `JSX.Element` is a better choice for type (can be text, one node, array of nodes)